releasenotes/2.1.0.md

```diff
@@ -0,0 +1,85 @@
+---
+layout: releasenotes
+title: Release Notes for Version 2.0.0
+date: 2016-11-15 12:00:00
```

Comment:
Put the release date here. It should be the same than the publish date for this page.